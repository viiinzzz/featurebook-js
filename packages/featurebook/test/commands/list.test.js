/* eslint-disable camelcase */
/* eslint-disable quotes */
const {
  given, when, then_wait_at_most, expect,
  node,
  one_minute,
} = require('../test.helpers');

given('I have a folder with .feature files', () => {
  const folder = '../features.simple';
  when('I run command ‘list’', () => {
    then_wait_at_most(one_minute)
      .that('it generates expected output to the console', async () => {
        const str = await node({
          program: 'bin/featurebook',
          options: `list ${folder}`,
        });
        expect(str).toMatchSnapshot();
      });
  });
});
