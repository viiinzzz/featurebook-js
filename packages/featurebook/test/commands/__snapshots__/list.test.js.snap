// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`I have a folder with .feature files I run command ‘list’ it generates expected output to the console 1`] = `
"featurebook v3.0.0-b.9 feature:

----------------------------------------------------------------------------------

B:/npmjs.published/featurebook-js/test/features.simple/Act 1/Act 1 Scene 1.feature

[1m[35m●[39m[22m [32mAct 1 Scene 1[39m

  [1m[35m›[39m[22m [32m‘tis but our fantasy[39m
    [1m[32m· GIVEN[39m[22m [32mMarcellus and Bernardo[39m
    [1m[32m· WHEN[39m[22m [32mHoratio comes[39m
    [1m[32m· THEN[39m[22m [32mMarcellus informs[39m

"
`;
